test/scripts/measures/2018/fixtures/test-quality-measures.csv

```diff
@@ -1,45 +1,76 @@
-,Diabetes: Hemoglobin A1c (HbA1c) Poor Control (>9%),CMS122v6,N/A,0059,001,Percentage of patients 18-75 years of age with diabetes who had hemoglobin A1c > 9.0% during the measurement period,Effective Clinical Care, Intermediate Outcome,National Committee for Quality Assurance,X,-,X,X,-,X,-,-,-,-,-,-,-,X,X,-,-,-,-,-,-,-,-,X,-,-,-,-,-,-,-,-,-,-,-,-,X,-,-,-,-,singlePerformanceRate,2017,NULL,,TRUE,TRUE,1,1,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-,Coronary Artery Disease (CAD): Beta-Blocker Therapy � Prior Myocardial Infarction (MI) or Left Ventricular Systolic Dysfunction (LVEF < 40%),CMS145v6,N/A,"
-0070
-",007,Percentage of patients aged 18 years and older with a diagnosis of coronary artery disease seen within a 12-month period who also have a prior MI OR a current or prior LVEF <40% who were prescribed beta-blocker therapy,Effective Clinical Care,process,Physician Consortium for Performance Improvement,-,-,X,-,-,X,-,-,X,-,-,-,-,X,X,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,multiPerformanceRate,2017,NULL,,FALSE,FALSE,NULL,2,2,weightedAverage,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-,Medication Reconciliation Post-Discharge,N/A,N/A,0097,046,"The percentage of discharges from any inpatient facility (e.g. hospital, skilled nursing facility, or rehabilitation facility) for patients 18 years and older of age seen within 30 days following discharge in the office by the physician, prescribing practitioner, registered nurse, or clinical pharmacist providing on-going care for whom the discharge medication list was reconciled with the current medication list in the outpatient medical record.
-This measure is reported as three rates stratified by age group:
-� Submission Criteria 1: 18-64 years of age
-� Submission Criteria 2: 65 years and older
-� Total Rate: All patients 18 years of age and older",Communication and Care Coordination,process,National Committee for Quality Assurance,X,-,-,X,-,X,-,-,-,-,-,-,-,-,-,-,-,X,-,-,-,-,-,-,-,-,-,-,-,X,-,-,-,-,-,-,X,-,-,-,-,multiPerformanceRate,2017,NULL,,TRUE,FALSE,3,3,NULL,overallStratumOnly,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,"metricType (required)
+
+
+
```

Comment:
not exactly expected, but i want to preserve the file as excel saves it instead of making any manual edits (which is also why i added the 0 padding code)